aries_cloudagent/registrar/models/job.py

```diff
@@ -43,7 +49,14 @@ def job_id(self) -> str:
     @property
     def record_value(self) -> dict:
         """Return values of record as dictionary."""
-        return {}
+        return {
+            "job_id": self.job_id,
+            "did": self.did,
+            "operation": self.operation,
+            "did_state": self.state,
```

Comment:
`did_state` should hold the entire DID state object. `state` reflects the `registration_state` from `did_state`.

In order to hold the DID state object, we'll have to serialize this value before returning it in `record_value`. On `__init__` we'll also have to deserialize the state from the serialized record.